backend/src/model/alert.py

```diff
@@ -10,5 +11,5 @@ class Alert(Base):
         primary_key=True, autoincrement=True, nullable=False
     )
     text: Mapped[str] = mapped_column(nullable=False)
-    start_datetime: Mapped[datetime.datetime] = mapped_column(nullable=False)
-    end_datetime: Mapped[datetime.datetime] = mapped_column(nullable=False)
+    start_datetime: Mapped[datetime.datetime] = mapped_column(DateTime, nullable=False)
```

Comment:
I think this should be Mapped[DateTime] -- yesterday when I ran some code that had Mapped[datetime.datetime], it did not work because the mapped_column is of type DateTime.